src/Examples/UnitModels/Reactors/stoichiometric_reactor_testing.ipynb

```diff
@@ -0,0 +1,1073 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.M101.reagent_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"].fix(58.0*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.reagent_feed.flow_mol_phase_comp[0, \"Liq\", \"water\"].fix(39.6*pyunits.mol/pyunits.s)  # calculated from 16.1 mol EO / cudm in stream\n",
+    "m.fs.M101.reagent_feed.flow_mol_phase_comp[0, \"Liq\", \"sulfuric_acid\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.reagent_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.reagent_feed.temperature.fix(298.15*pyunits.K)\n",
+    "m.fs.M101.reagent_feed.pressure.fix(1e5*pyunits.Pa)\n",
+    "\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"water\"].fix(200*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"sulfuric_acid\"].fix(0.334*pyunits.mol/pyunits.s)  # calculated from 0.9 wt% SA in stream\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.temperature.fix(298.15*pyunits.K)\n",
+    "m.fs.M101.catalyst_feed.pressure.fix(1e5*pyunits.Pa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fixing unit model specifications\n",
+    "\n",
+    "Now that we have fixed our inlet feed conditions, we will now be fixing the operating conditions for the unit models in the flowsheet. Let us fix the outlet temperature of H101 to 328.15 K. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.H101.outlet.temperature.fix(328.15*pyunits.K)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will need to specify both initial conversion and heat duty values (these are the only two free variables to choose from). Since heat duty and the outlet reactor temperature are interdependent, we can choose to specify  this quantity instead. While the reaction kinetic parameters exist in the property package, we also do not need to add a rate constant expression since generation is explicitly defined through the conversion/yield. Note that our initial problem will solve with zero *temperature change* but will be infeasible with zero *heat duty*; this is due to the heat of reaction enforced by allowing heat transfer and mandating a non-zero conversion."
```

Comment:
I wonder if it is worth mentioning that there are aa number of ways the reaction extent could be defined; e.g. conversion or yield.

Reply:
Possibly, conversion is typically more convenient since it is independent of the reaction stoichiometry. There are multiple ways to specify yield (mol EG/mol EO, kg EG/kg EO, or volumetric units) while conversion is more straightforward always being on 0-100% for a given basis. I should also emphasize that conversion is chosen with respect to a specific reactant.

Reply:
Not quite - conversion also depends on the stoichiometric basis chosen, as it can be specified with respect to any of the reactants as well, and as reactants are not always fed stoichiometrically a 100% conversion of one reactant is not always equivalent to a 100% conversion of the other (although conventionally we  would use the limiting reagent as the basis, it is just as valid to do it another way).

However, my main point here is that we should highlight that IDAES supports any of these approaches and bases for specifying the reaction extent (including directly setting the extent), and that we are going to use conversion of species X for this example.

Reply:
The section has been updated with the details discussed above.